database/migrations/2022_04_29_184818_create_jobs_table.php

```diff
@@ -13,13 +13,13 @@
      */
     public function up()
     {
-        Schema::create('personal_access_tokens', function (Blueprint $table) {
+        Schema::create('jobs', function (Blueprint $table) {
             $table->id();
-            $table->morphs('tokenable');
-            $table->string('name');
-            $table->string('token', 64)->unique();
-            $table->text('abilities')->nullable();
-            $table->timestamp('last_used_at')->nullable();
+            $table->foreignId('employers_id')->references('id')->on('employers')->onUpdate('cascade')->onDelete('cascade');
+            $table->string('keywords');
+            $table->string('title');
+            $table->string('location');
+            $table->string('type');
             $table->timestamps();
```

Comment:
Have you considered to use “softDeletes” for your models? Is that so you should include the proper field here and the trait on the models, check the docs

Reply:
I've read the docs about soft deletes now, thanks for pointing it out, but I don't think I need it at this. point.